docker/all-in-one/shutdown_event_listener.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+# This script is used to make sure the latest LSN checkpoint is persisted remotely
+#  before the container is stopped. It is triggered every 60 seconds, whichever comes first.
```

Comment:
Perhaps I'm misreading the script, but how does any invocation terminate? Or is it a single instance that's spawned, and gets new events fed in over stdin as they occur?

Reply:
It's an ongoing script which supervisord keeps alive then feeds it events, yes.